src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Blocks/EmptyState/product.html.twig

```diff
@@ -0,0 +1,53 @@
+{#**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+<div class="text-center showcase-list-card">
+  <img class="img-responsive mt-3 img-rtl" src="{{ asset('themes/new-theme/img/empty_state/catalog.png') }}">
+
+  <p class="mt-4 showcase-list-card__header">
+    {{ "Let's start building your catalog!"|trans({}, 'Admin.Catalog.Feature') }}
+  </p>
+
+  <p class="mx-auto showcase-list-card__message">
+    {{ 'Building your catalog is the key step; present your brand here and make visitors feel like shopping on your store. Our advice? Pay extra attention to your product photography, neat pictures could increase your conversion rate by 35%!'|trans({}, 'Admin.Catalog.Feature') }}
```

Comment:
```suggestion
    {{ 'Building your catalog is the key step; present your brand here and make visitors feel like shopping on your store. Our advice? Pay extra attention to your product's images, neat pictures could increase your conversion rate by 35%!'|trans({}, 'Admin.Catalog.Feature') }}
```

Reply:
In the end, it looks more consistent with the terminology of the page, see below:

<img width="920" alt="Capture d’écran 2020-01-30 à 14 41 55" src="https://user-images.githubusercontent.com/32565890/73454652-cb083e00-436e-11ea-9050-bf419980eb63.png">